### ISSUE TITLE
NVDA is unable to find local voices

### ISSUE THREAD

**Issue by austek** [ZirekHQ/dengjen-nvda#8]
Mirrored from upstream issue [mush42/sonata-nvda#67](https://github.com/mush42/sonata-nvda/issues/67) (originally reported by @Violet-Iolite on 2025-10-07).

Tracked here so it can be triaged on the maintenance fork — see PR #72 / the [NVDA Add-ons list announcement](https://nvda-addons.groups.io/g/nvda-addons/message/27636) for context.

---

Note: This issue might be a problem related to the fact this is my work computer and not the add on's problem. It might also be my fault which is why I came here to ask for advice.

So, I wanted to use Piper voices in my Windows work laptop. I decided to install the Sonata add on for NVDA which did install without issues. 
The automatic download of voices isn't working, but I'm not surprised because of the web restrictions my laptop has. So, I decided to try the local voices. I got the files for the voices on Hugging Face, I even made a .tar.gz file (Tar GNU with GZIP compression) but when I choose the local download, even though it can find the compressed folder and open it it can't find the .onx and .onx.jason files. Because of that I can't install the voices.
Could it be I'm not formatting it correctly, that I need something else, or could my laptop be blocking the access these files somehow? It did find and open the folder the files were on, although it only showed it was empty despite the files being there.

Thank you for this project anyway!
